roles/openshift_prometheus/templates/prometheus.rules.j2

```diff
@@ -0,0 +1,13 @@
+groups:
+- name: example-rules
+  interval: 30s # defaults to global interval
+  rules:
+  - alert: Node Down
+    expr: up{job="kubernetes-nodes"} == 0
+    annotations:
+      miqTarget: "ContainerNode"
+      severity: "HIGH"
+      message: "{{ '{{' }}{{ '$labels.instance' }}{{ '}}' }} is down"
+{% if openshift_prometheus_external_rules is defined and openshift_prometheus_external_rules is not none %}
+  - {{ openshift_prometheus_external_rules }}
+{% endif %}
```

Comment:
@simon3z this is the way external alert rules are handled. The user can override the `openshift_prometheus_external_rules` variable with additional rule definition and those will be added to the configmap